R/sensivitityTimeProfiles.R

```diff
@@ -0,0 +1,134 @@
+#' @name sensitivityTimeProfiles
+#' @title Create a concentration-time profile plot
+#'
+#' @inheritParams sensitivitySpiderPlot
+#' @inheritParams colorspace::scale_color_continuous_qualitative
+#'
+#' @import ggplot2
+#' @import dplyr
+#' @import colorspace
+#'
+#' @examples
+#'
+#' simPath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
+#' simulation <- loadSimulation(simPath)
+#' outputPaths <- "Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)"
+#' parameterPaths <- c(
+#'   "Aciclovir|Lipophilicity",
+#'   "Applications|IV 250mg 10min|Application_1|ProtocolSchemaItem|Dose",
+#'   "Neighborhoods|Kidney_pls_Kidney_ur|Aciclovir|Glomerular Filtration-GFR|GFR fraction"
+#' )
+#'
+#' # extract the results into a list of dataframes
+#' results <- sensitivityCalculation(
+#'   simulation = simulation,
+#'   outputPaths = outputPaths,
+#'   parameterPaths = parameterPaths
+#' )
+#'
+#' # print plots
+#' sensitivityTimeProfiles(results)
+#'
+#' # print and save plots
+#' if (FALSE) {
+#'   sensitivityTimeProfiles(
+#'     results,
+#'     savePlots = TRUE,
+#'     units = "in",
+#'     height = 6,
+#'     width = 12
+#'   )
+#' }
+#'
+#' @export
+sensitivityTimeProfiles <- function(sensitivityCalculation,
```

Comment:
What's `units` here? Somehow I have no documentation for this function.

If these are the units for widht/height - it is better to rename the argument, as in the OSP context we usually think about units of the simulation output (µmol, h).

Reply:
No, it was units for plot dimensions. 
I have removed this argument. Height and width for saving plots should always be specified in inches now.